prymer/offtarget/bwa.py

```diff
@@ -297,10 +306,9 @@ def __init__(
 
     def __signal_bwa(self) -> None:
         """Signals BWA to process the queries"""
-        for _ in range(3):
-            self._subprocess.stdin.flush()
-            self._subprocess.stdin.write("\n\n")
-            self._subprocess.stdin.flush()
+        self._subprocess.stdin.flush()
+        self._subprocess.stdin.write("\n" * 16)
```

Comment:
**suggestion** Leave a comment/breadcrumb to document the reason for this and the choice of 16